sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -365,16 +365,16 @@ export const logger: import("@azure/logger").AzureLogger;
 export type MergeCertificateOptions = coreHttp.OperationOptions;
 
 // @public
-export interface ParsedKeyVaultCertificatesIdentifier {
+export function parseCertificateId(id: string): ParsedCertificateId;
+
+// @public
+export interface ParsedCertificateId {
```

Comment:
How does this relate to a `KeyVaultCertificate`, though? Can't you just drop the "Parsed" prefix if it's too long otherwise? .NET and JS could differ in "Id" vs. "Identifier", but I feel this creates a much larger discrepency between languages.

It's only a preview, sure, but there are a small enough number of changes we should settle on idiomatic consistencies ASAP.

/cc @iscai-msft @chlowell @g2vinay

Reply:
I could make it `KeyVaultIdentifier` instead of `Parsed*`. That sounds reasonable to me.

Reply:
Then we would need to make this specific to Certificates, since we would be exporting three. So, `KeyhVaultCertificateIdentifier`?

Reply:
If `KeyVaultRoleAssignmentProperties` and `KeyVaultRoleAssignmentPropertiesWithScope` satisfies the guideline that a service prefix should be used if the absence of it results in a generic term with no hints as to which service it is related to, then probably `KeyVaultCertificateIdentifier` does too?, from [this other thread](https://github.com/Azure/azure-sdk-for-js/pull/10815#discussion_r478397241).